programs/leap-util/actions_blocklog.cpp

```diff
@@ -0,0 +1,339 @@
+#include "actions_blocklog.hpp"
+#include <eosio/chain/abi_serializer.hpp>
+#include <eosio/chain/block_log.hpp>
+#include <eosio/chain/config.hpp>
+#include <eosio/chain/fork_database.hpp>
+#include <memory>
+
+#include <fc/bitutil.hpp>
+#include <fc/filesystem.hpp>
+#include <fc/io/json.hpp>
+#include <fc/variant.hpp>
+
+#include <boost/exception/diagnostic_information.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/filesystem/path.hpp>
+#include <boost/program_options.hpp>
+
+#include <chrono>
+
+#ifndef _WIN32
+#define FOPEN(p, m) fopen(p, m)
+#else
+#define CAT(s1, s2) s1##s2
+#define PREL(s) CAT(L, s)
+#define FOPEN(p, m) _wfopen(p, PREL(m))
+#endif
+
+using namespace eosio::chain;
+namespace bfs = boost::filesystem;
+namespace bpo = boost::program_options;
+using bpo::options_description;
+using bpo::variables_map;
+
+struct report_time {
+   report_time(std::string desc)
+       : _start(std::chrono::high_resolution_clock::now()), _desc(desc) {
+   }
+
+   void report() {
+      const auto duration = std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::high_resolution_clock::now() - _start).count() / 1000;
+      ilog("eosio-blocklog - ${desc} took ${t} msec", ("desc", _desc)("t", duration));
+   }
+
+   const std::chrono::high_resolution_clock::time_point _start;
+   const std::string _desc;
+};
+
+
+void BlocklogActions::setup(CLI::App& app) {
```

Comment:
Here, and throughout, use snake_case instead of CamelCase.

Reply:
Done, everything snake_cased, thank you!